cranelift-wasm/src/code_translator.rs

```diff
@@ -928,6 +928,16 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             // the v128.const is typed in CLIF as a I8x16 but raw_bitcast to a different type before use
             state.push1(value)
         }
+        Operator::V128Load {
+            memarg: MemoryImmediate { offset, .. },
```

Comment:
For consistency, can you please put this one next to the other load operators (resp. store operators), and make it more apparent that the `flags` immediate isn't used right now? (as is done for other memory operators)